client/components/ScopeDropdown/ScopeDropdown.tsx

```diff
@@ -209,7 +216,9 @@ const ScopeDropdown = (props: Props) => {
 													mode: 'layout',
 												})}
 											>
-												<Icon icon={pubPubIcons.layout} iconSize={12} />
+												<Tooltip content="Visit collection">
```

Comment:
"Collection" should always be capitalized in user-facing text. Also, this button doesn't (or at least, shouldn't) take you to the Collection itself, but to the Collection's Layout tab. So it should just say "Layout".